package.json

```diff
@@ -21,7 +23,6 @@
     "playkit-dev-ott:start": "playkit-dev start ott",
     "playkit-dev:stop": "playkit-dev stop",
     "playkit-rel": "playkit-rel start",
-    "precommit": "lint-staged",
```

Comment:
Why removing linting?

Reply:
![image](https://user-images.githubusercontent.com/5623935/51604469-2f4b2300-1f0d-11e9-9b0f-bae8cc23fb90.png)

Reply:
Mayyybe just fix the errors? :D

Reply:
reverted